angular.json

```diff
@@ -34,6 +34,19 @@
           "configurations": {
             "ie": {
               "tsConfig": "src/tsconfig-ie.spec.json"
```

Comment:
Can be removed!

Reply:
You still need it if you want to run unit tests with `ie`